scripts/data_transfer/copy_sampleid_keys.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil cp "gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/OneK1K_CPG_IDs.tsv" gs://cpg-tob-wgs-main/scrna-seq/grch38_association_files/
```

Comment:
NL at end of files, otherwise some linters complain

```suggestion
gsutil cp "gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/OneK1K_CPG_IDs.tsv" gs://cpg-tob-wgs-main/scrna-seq/grch38_association_files/

```